server/http.js

```diff
@@ -1,14 +1,38 @@
 
-var express = require('express'),
-    Placeholder = require('../Placeholder.js');
+/**
+  The http server improves performance on multicore machines by using the
+  node core 'cluster' module to fork worker processes.
 
-const morgan = require( 'morgan' );
-const logger = require('pelias-logger').get('interpolation');
-const through = require( 'through2' );
+  The default setting is to use all available CPUs, this will spawn 32 child
+  processes on a 32 core machine.
+
+  If you would like to disable this feature (maybe because you are running
+  inside a container) then you can do so by setting the env var CPUS=1
+
+  You may also specify exactly how many child processes you would like to 
+  spawn by setting the env var to a numeric value >1, eg CPUS=4
+
+  If the CPUS env var is set less than 1 or greater than os.cpus().length
+  then the default setting will be used (using all available cores).
+**/
+
+const os = require('os');
+const morgan = require('morgan');
+const express = require('express');
+const cluster = require('cluster');
+const through = require('through2');
 const _ = require('lodash');
 
-// optionally override port using env var
+const Placeholder = require('../Placeholder.js');
+const logger = require('pelias-logger').get('interpolation');
```

Comment:
this should be `placeholder`, not `interpolation`, right?

Reply:
good catch, this was introduced in https://github.com/pelias/placeholder/commit/8d83744e69dda00673412d2890f32cd5446899fb, I will add another commit to fix it in this PR